code/variadic/README.md

```diff
@@ -0,0 +1,22 @@
+
+## Instructions
+
+In this exercise, we will take the `tuple` implementation from the slides
+and extend it step by step to make the example code in `main` compile.
+
+Feel free to comment out parts of main to test parts of your implementation sooner.
+All instructions are given in more detail in the source code as well.
+
+This example neglects const correctness so there is less code to write.
+Making it const correct would require adding several overloads with almost the same code as the non-const version.
+You can try this is a bonus exercise at the end!
+
+1. make `tuple` default construct
```

Comment:
For me this doesn't parse.
- *a* default construct?
- default construct*able*
- Ensure that `tuple` has a default constructor

By looking at the solution, the best is presumably:
```suggestion
1. Ask the compiler to generate a default constructor for `tuple`
```

Reply:
I would say that the instructions should be much more detailed. I struggle to understand what to do. More suggestions to follow soon

Reply:
And the statement means: "Make tuple<T...> t; compile". So, make `tuple` default constructible would be the best of your wordings.

Reply:
Yes, because this is where they should add some code as detailed inside `main` :)

Reply:
Ah, OK. I didn't manage to read that far down before the meeting. Now continuing ...

Reply:
I rephrased `make tuple default construct` to `make tuple default-constructible` and pointed out more strongly in the README.md that the detailed instructions are to be found in `main`.